python/openmldb/dbapi/dbapi.py

```diff
@@ -456,12 +454,15 @@ def get_resultset_schema(self):
 
 class Connection(object):
 
-    def __init__(self, db, zk, zkPath):
+    def __init__(self, db,is_cluster_mode=True, *args):
```

Comment:
Format the code and add spaces between parameters.

Reply:
updated.